internal/schema/diff.go

```diff
@@ -57,6 +57,9 @@ func calculateDiff(past []*Resource, current []*Resource) []*Resource {
 func diffResourcesByKey(resourceKey string, pastResMap, currentResMap map[string]*Resource) (bool, *Resource) {
 	past, pastOk := pastResMap[resourceKey]
 	current, currentOk := currentResMap[resourceKey]
+	if current == nil && past == nil {
+		return false, nil
```

Comment:
Should we add logging here to output the resourceKey so we can debug if this happens again?

Reply:
That would be great. However, we can't debug it just be this log and if the user provides its config then we can reproduce it. But I think it's better to add logs here and in the same location in the `diffCostComponentsByKey` function.

Reply:
@aliscott Added some debug info. Do we need more data in the log?